python/tank/authentication/ui/login_dialog.py

```diff
@@ -119,6 +119,13 @@ def setupUi(self, LoginDialog):
         self.button_layout.setSpacing(10)
         self.button_layout.setContentsMargins(0, -1, -1, -1)
         self.button_layout.setObjectName("button_layout")
+        self.links = QtGui.QVBoxLayout()
+        self.links.setObjectName("links")
```

Comment:
Might be worth posting a video on #shotgun-toolkit in Slack as well so people get a feel of the flow.

+10000 if it's an animated gif on GitHub. ;)